app/views/slices/firstParagraph.php

```diff
@@ -1,29 +1,29 @@
 <?php
+use Prismic\Dom\RichText;
 
 // Get all the slices of the post
-$sliceZone = $post->getSliceZone('post.body');
+$slices = $post->data->body;
 $firstParagraph = "";
 
 // Loop through each slice to find the first text slice
-foreach ($sliceZone->getSlices() as $slice) {
-  if ($slice->getSliceType() == "text") {
-    $firstParagraph = $slice->getPrimary()->getStructuredText('text')->getFirstParagraph();
+foreach ($slices as $slice) {
+  if ($slice->slice_type == "text") {
+    $firstText = RichText::asHtml($slice->primary->text, $linkResolver);
```

Comment:
Is there a reason you changed this to `$firstText`? I'm fine with that, but just make sure to update the `$firstParagraph` variable above to give this all a default state.